lib/pages/AboutUsPage.dart/AboutUs.dart

```diff
@@ -31,10 +33,25 @@ class _AboutUsState extends State<AboutUs> {
             SizedBox(
               height: 20,
             ),
-            Text(
-              "Thank you for using KIDE, the KIIT guide by the students, for the students! We're a small team of Flutter Developers who decided to take the initiative to create an app which could serve as a hub for all the day-to-day student tasks as well as general tasks related to our esteemed university. We hope that our work reflects this ideology and you get to enjoy a seamless user experience with minimal issues. However, we'd love to hear from you. Feel free write your suggestions and queries to us at - kide.kiit@gmail.com",
-              style: TextStyle(fontSize: 20, fontFamily: 'EncodeSans'),
-            )
+            RichText(
+              text: TextSpan(
+                children: [
+                  TextSpan(
+                    text: "Thank you for using KIDE, the KIIT guide by the students, for the students! " +
+                        "We're a small team of Flutter Developers who decided to take the initiative to create an app which could serve as a hub for all the day-to-day student " +
+                        "tasks as well as general tasks related to our esteemed university. We hope that our work reflects " +
+                        "this ideology and you get to enjoy a seamless user experience with minimal issues. However, " +
+                        "we'd love to hear from you. Feel free write your suggestions and queries to us at - ",
+                  ),
+                  TextSpan(
+                    text: 'kide.kiit@gmail.com',
+                    style: TextStyle(decoration: TextDecoration.underline, color: Colors.blue),
+                    recognizer: TapGestureRecognizer()..onTap = () => launch('mailto:kide.kiit@gmail.com'),
```

Comment:
A better practice would be to check `canLaunch()` before the launch. Rest of the work seems good 👍

Reply:
Should i resend a pull request with improvements...

Reply:
No need for a new PR. You can just commit your changes to this branch and they would reflect here.

Reply:
> A better practice would be to check `canLaunch()` before the launch. Rest of the work seems good 👍

Good catch @emem365, @rrkas we will be awaiting these changes.

Reply:
Sorry sir. I am a beginner. I didn't know that. I'm on it.

Reply:
Sue @rrkas, dont worry - this is the best learning opportunity for beginners like you. You will certainly be benefitted through this.

Reply:
committed and pushed. please check.